docs/farming-&-staking/farming/common-problems.mdx

```diff
@@ -0,0 +1,129 @@
+<summary><small>WARN Consensus: sc_proof_of_time::source: Proof of time chain reorg happened from_next_slot=&lt;block&gt; to_next_slot=&lt;block&gt;</small></summary>
+
+
+🟠 This situation arises when two or more valid blocks are created at the same height in the blockchain. Initially, network nodes may disagree on which block to add to the chain, resulting in multiple temporary forks. Once this disagreement is resolved, the blocks from the discarded fork become orphaned, and the chain reorganizes to incorporate the blocks from the winning fork.
+
+This should resolve on its own.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+<details>
+<summary><small>WARN Consensus: sc_proof_of_time::source: Proof of time chain was extended from block import from_next_slot=&lt;block&gt; to_next_slot=&lt;block&gt;</small></summary>
+
+
+🟠 This warning message should not affect your rewards or the operation of your node. It's just a log message indicating the normal operation of the network.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+
+<TabItem value="farmer" label="Farmer">
+
+<details>
+<summary><small>ERROR subspace_farmer::farmer_cache: Failed to store piece in cache, there was no space piece_index=&lt;piece&gt;</small></summary>
+
+
+🔴 This indicates that there is not enough space in the cache to store a new piece of data.
+
+The drive where you are storing your plot is out of space. Reduce the plot size and restart the farmer. Additionally, ensure that you are saving the plot to the correct drive and that it is properly mounted.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+
+<TabItem value="farming-cluster" label="Farming Cluster">
+
+<details>
+<summary><small>WARN \{farm_index=3}:\{sector_index=1036}: subspace_farmer::single_disk_farm::plotting: Failed to plot sector, retrying in 1s error=Low-level plotting error: Received only &lt;sector&gt; sector bytes out of &lt;bytes&gt; expected bytes</small></summary>
+
+
+🟠 This is a known issue that is actively being investigated. To help mitigate these errors, run your NATS component on the same system as your Controller, Cache, and Plotter components.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+</Tabs>
+
+## Unexpected Behavior
+
+These unexpected behaviors are not linked to specific components. In this section, we will outline common issues and provide any available remediation steps to address them.
+
+:::caution[Handling No Output and Missing Error Codes]
+
+If you encounter a situation where the node produces no output and does not display any error code, it is likely that you simply need to install the latest version of the Visual C++ Redistributable package, which can be found [here](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).
+
+:::
+
+:::tip[Farmer is stuck on plotting and has not made any progress for several hours.]
```

Comment:
Are you also trying to follow the red / yellow / green pattern, depending on the severity? This sections reads a bit weird with some of the errors using caution's ! and some tip's lightbulb

Reply:
I agree with this assessment and will update to the same style as the Program Issues (previously components) section.